conftest.py

```diff
@@ -83,3 +91,15 @@ def pytest_sessionfinish(session, exitstatus):
 
     reporter = session.config.pluginmanager.get_plugin("terminalreporter")
     reporter.summary_stats()
+
+
+@pytest.fixture(scope="session")
+def s3_bucket_name_wisdom(pytestconfig: pytest.Config) -> Optional[str]:
```

Comment:
```suggestion
def s3_bucket_name_wisdom(pytestconfig: pytest.Config) -> str:
```
will always return because if not we raise

please rename to something like `models_s3_bucket_name` (and not include `wisdom`)